website/source/docs/guides/monitoring-telegraf.html.md

```diff
@@ -227,18 +278,30 @@ you will need to apply a function such as [non_negative_difference][].
 | `diskio.read_bytes` | Bytes read from each block device. |
 | `diskio.write_bytes` | Bytes written to each block device. |
 
-**Why they're important:** If the Consul host is writing a lot of data to disk, such as under high volume workloads, there may be frequent major I/O spikes during leader elections. This is because under heavy load,
-Consul is checkpointing Raft snapshots to disk frequently.
+**Why they're important:** If the Consul host is writing a lot of data to disk,
+such as under high volume workloads, there may be frequent major I/O spikes
+during leader elections. This is because under heavy load, Consul is
+checkpointing Raft snapshots to disk frequently.
+
+It may also be caused by Consul having debug/trace logging enabled in
+production, which can impact performance.
+
+Too much disk I/O can cause the rest of the system to slow down or become
+unavailable, as the kernel spends all its time waiting for I/O to complete.
 
-It may also be caused by Consul having debug/trace logging enabled in production, which can impact performance.
+**What to look for:** Sudden large changes to the `diskio` metrics (greater than
+50% deviation from baseline, or more than 3 standard deviations from baseline).
 
-Too much disk I/O can cause the rest of the system to slow down or become unavailable, as the kernel spends all its time waiting for I/O to complete.
+**NOTE:** The `diskio` metrics are counters, so in order to calculate rates
+(such as bytes/second), you will need to apply a function such as
+[non_negative_difference][].
 
-**What to look for:** Sudden large changes to the `diskio` metrics (greater than 50% deviation from baseline,
-or more than 3 standard deviations from baseline).
+## Summary
 
-**NOTE:** The `diskio` metrics are counters, so in order to calculate rates (such as bytes/second),
-you will need to apply a function such as [non_negative_difference][].
+In this guide you learned how to set up Telegraf with Consul to collect metrics,
+and considered your options for visualizing, aggregating, and alerting on those
+metrics. To learn about other factors (in addition to monitoring) that you
+should consider when running Consul in production, see the [Production Checklist][prod-checklist].
 
 [non_negative_difference]: https://docs.influxdata.com/influxdb/v1.5/query_language/functions/#non-negative-difference
```

Comment:
We should discuss this link format in our style guide. Personally, I'm not a fan and tend to add links in the text of the doc.

Reply:
I like it because it cuts down on link fixes when you refer to the same URL more than once. But it works for me because I always build the site to review. It can be had to check that the links are right if you're reviewing raw markdown. I could see the value of either way, but we should definitely be consistent 🙂

Reply:
I think I'd also prefer not to link to the same URL more than once  either :) Leaving it for now is probably fine but we should revisit the style guide next quarter!